salt/metalk8s/addons/logging/fluent-bit/deployed/post-downgrade.sls

```diff
@@ -0,0 +1,31 @@
+{%- set dest_version = pillar.metalk8s.cluster_version %}
+
+{#- Some objects has been removed/renamed with the new fluent-bit chart,
+    so let's cleanup new ones
+    NOTE: This logic can be removed in `development/124.0` #}
+
+{%- if salt.pkg.version_cmp(dest_version, '123.0.0') == -1 %}
+
+Cleanup new fluent-bit ServiceMonitor:
+    metalk8s_kubernetes.object_absent:
+        - apiVersion: monitoring.coreos.com/v1
+        - kind: ServiceMonitor
+        - name: fluent-bit
+        - namespace: metalk8s-logging
+
+Cleanup new fluent-bit Service:
+    metalk8s_kubernetes.object_absent:
+        - apiVersion: v1
+        - kind: Service
+        - name: fluent-bit
+        - namespace: metalk8s-logging
+
+{# This one just get renamed from `fluent-bit-clusterrole` to `fluent-bit` #}
+Cleanup new fluent-bit ClusterRole:
```

Comment:
There was also a `fluent-bit-clusterrolebinding` apparently

Reply:
Oh good catch I miss this one :+1:

Reply:
Fixed